BookingsAPI/Bookings.DAL/Commands/DeleteLinkedParticipantCommand.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Threading.Tasks;
+using Bookings.DAL.Commands.Core;
+using Bookings.DAL.Exceptions;
+using Microsoft.EntityFrameworkCore;
+
+namespace Bookings.DAL.Commands
+{
+    public class DeleteLinkedParticipantCommand : ICommand
```

Comment:
Do we still need to delete links or delete the participant?

Reply:
We may have a situation in the future where the linked participant needs to be deleted but the participant would need to stay?